less/home-page.less

```diff
@@ -58,4 +63,18 @@
       margin-bottom: 30px;
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+
+@media (max-width: 500px) {
+  body {
```

Comment:
targeting `.content-section, .content-pick` should get them to stack vertically.